### PR TITLE
Fix staking APY logic

### PR DESCRIPTION
**Pull request by shahthepro**

### CODE REVIEW

File: src/governance/sections/StepByStep.tsx [hunk 1]
```diff
@@ -89,7 +83,7 @@ const StepByStep = ({ sectionOverrideCss }: StepByStepProps) => {
             {
               {
                 1: <Step1 />,
-                2: <Step2 stakingApy={stakingApy} />,
+                2: <Step2 stakingApy={stakingAPY} />,
```
Comment (rolandpo) [inline src/governance/sections/StepByStep.tsx:86]: think there's a breaking typo here, stakingApy -> stakingAPY

**Review by rolandpo — APPROVED** [review]

File: src/governance/sections/StepByStep.tsx (hunk 1 above)
Comment (shahthepro) [reply src/governance/sections/StepByStep.tsx:86]: I changed the case in all places. So it shouldn't break anything :)

Comment (rolandpo) [reply src/governance/sections/StepByStep.tsx:86]: the lowercase doesn't match the one inside Step2, no?

Comment (shahthepro) [reply src/governance/sections/StepByStep.tsx:86]: Ah, my bad, I missed it somehow, fixing it now. Thanks for catching it

**Pull request merged**